pkgs/top-level/aliases.nix

```diff
@@ -1039,6 +1039,7 @@ mapAliases ({
   prismlauncher-qt5 = throw "'prismlauncher-qt5' has been removed from nixpkgs. Please use 'prismlauncher'"; # Added 2024-04-20
   prismlauncher-qt5-unwrapped = throw "'prismlauncher-qt5-unwrapped' has been removed from nixpkgs. Please use 'prismlauncher-unwrapped'"; # Added 2024-04-20
   privacyidea = throw "privacyidea has been removed from nixpkgs"; # Added 2023-10-31
+  probe-rs = throw "probe-rs binaries were moved to the probe-rs-tools package"; # Added 2024-05-23
```

Comment:
```suggestion
  probe-rs = probe-rs-tools; # Added 2024-05-23
```

Since this change isn't major.

Reply:
Makes sense, done!  Thanks for the review.